Building fails with unprovided dependency

Building fails with
```
[ERROR] Cannot resolve target definition:
[ERROR]   Software being installed: org.eclipse.jdt.ls.core 1.34.0.202403160337
[ERROR]   Missing requirement: org.eclipse.jdt.ls.core 1.34.0.202403160337 requires 'osgi.bundle; org.eclipse.lsp4j [0.22.0.0,0.23.0)' but it could not be found
[ERROR] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Java Debug Server for Visual Studio Code :: Parent 0.51.1:
[INFO] 
[INFO] Java Debug Server for Visual Studio Code :: Parent . SUCCESS [  0.002 s]
[INFO] Java Debug Server for Visual Studio Code :: Debugger Core SUCCESS [  2.655 s]
[INFO] Java Debug Server for Visual Studio Code :: Debugger Plugin FAILURE [  1.402 s]
[INFO] Java Debug Server for Visual Studio Code :: P2 Update site SKIPPED
[INFO] Java Debug Server for Visual Studio Code :: Target Platform SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  6.424 s
[INFO] Finished at: 2024-03-16T20:16:24+01:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to resolve target definition file:/home/ben/testNeovim/java-debug/com.microsoft.java.debug.target/com.microsoft.java.debug.tp.target: See log for details
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
```

#### Same problem here


```
[INFO] Downloading from p2: https://download.eclipse.org/lsp4j/updates/releases/0.21.0/p2.index
[INFO] Downloaded from p2: https://download.eclipse.org/lsp4j/updates/releases/0.21.0/p2.index (172 bytes at 0 bytes/s)
[ERROR] Cannot resolve target definition:
[ERROR]   Software being installed: org.eclipse.jdt.ls.core 1.34.0.202403160337
[ERROR]   Missing requirement: org.eclipse.jdt.ls.core 1.34.0.202403160337 requires 'osgi.bundle; org.eclipse.lsp4j [0.22.0.0,0.23.0)' but it could not be found
[ERROR] 
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Java Debug Server for Visual Studio Code :: Parent 0.51.1:
[INFO] 
[INFO] Java Debug Server for Visual Studio Code :: Parent . SUCCESS [  0.462 s]
[INFO] Java Debug Server for Visual Studio Code :: Debugger Core SUCCESS [ 36.707 s]
[INFO] Java Debug Server for Visual Studio Code :: Debugger Plugin FAILURE [ 26.814 s]
[INFO] Java Debug Server for Visual Studio Code :: P2 Update site SKIPPED
[INFO] Java Debug Server for Visual Studio Code :: Target Platform SKIPPED
```

#### OS

Pop!_OS 22.04 LTS x86_64

#### git log

```
commit 98db4b16262488c5351a922f5781f7b95e1290d4 (HEAD -> main, origin/main, origin/HEAD)
Author: Jinbo Wang <jinbwan@microsoft.com>
Date:   Thu Mar 14 13:58:54 2024 +0800

    Update com.microsoft.java.debug.tp.target to 4.31 release (#538)
```

--- a/com.microsoft.java.debug.target/com.microsoft.java.debug.tp.target
+++ b/com.microsoft.java.debug.target/com.microsoft.java.debug.tp.target
@@ -31,7 +31,7 @@
             <repository location="https://download.eclipse.org/jdtls/snapshots/repository/latest/"/>
         </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-           <repository location="https://download.eclipse.org/lsp4j/updates/releases/0.21.0/"/>
+           <repository location="https://download.eclipse.org/lsp4j/updates/releases/0.22.0/"/>
            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.0.0"/>
         </location>
     </locations>

